vocabulary-draft/continuous-integration-pipeline-events.md

```diff
@@ -0,0 +1,26 @@
+# Continuous Integration Events
+
+These events are related to **Continuous Integration(CI)** activities. CI usually include activities such as building, testing, packaging and releasing software artifacts. 
+
+The following events represent concrete Tasks that are associated with the execution of CI pipelines:
+
+- **Build Queued**: a Build task has been queued, this build process usually is in charge of producing a binary from source code
+- **Build Started**: a Build task has started 
+- **Build Finished**: a Build task has finished, the event will contain the finished status, success, error or failure
+
+The following Test events are defined in two separate categories **Test Case** and **Test Suite**. A **Test Case** is the smallest unit of testing that the user wants to track. A **Test Suite** is a collection of test case executions and/or other test suite executions. **Test Cases** executed, and Test Suites are for grouping purposes. For this reason, **Test Cases** can be queued. 
+
+- **Test Case Queued**: a Test task has been queued, and it is waiting to be started
+- **Test Case Started**: a Test task has started
+- **Test Case Finished**: a Test task has finished, the event will contain the finished status, success, error or failure
+- **Test Suite Started**: a Test task has started
```

Comment:
Description should be updated, also for Test Suite Finished below.

Suggestion:
a test suite has started
a test suite has finished, the event will contain the finished status, success, error or failure

Reply:
@erkist thanks for catching that up.. fixed